pkg/config/config.go

```diff
@@ -38,6 +38,7 @@ type Config struct {
 	InsecureRegistry bool     `toml:"insecure_registry"`
 	HostsDir         []string `toml:"hosts_dir"`
 	Experimental     bool     `toml:"experimental"`
+	HostGatewayIP    string   `toml:"host_gateway_ip"`
```

Comment:
```suggestion
	HostGateway    *net.IP   `toml:"host_gateway_ip"`
```

Reply:
(Can remain as a `string` for ease of implementation)

Reply:
I see for any other configs, the variable name is exactly same as the field name in toml. Is it good to make it special as "HostGateway" instead of "HostGatewayIP"?

Reply:
Sorry, just forgot to modify the toml tag in my comment above

Reply:
`host-gateway-ip` is the key string in Docker config. https://github.com/moby/moby/blob/master/cmd/dockerd/config.go#L39

Should we keep Docker compatibiilty as much as possible or the config can be exempted?

Reply:
Ah, `host-gateway-ip` is the right choice then, sorry